### ISSUE TITLE
implement :split :vsplit

### ISSUE THREAD

**Issue by 257** [brookhong/Surfingkeys#960]
i use this all the time in vi; helps with inspecting the same page multiple times, for example.

**Comment by brookhong** [comment]
What do you want to do with `split`?

Is `yt` what you need?

**Comment by 257** [comment]
> Is `yt` what you need?

not at all, in vim, ```split``` with no argument will split the buffer (vertical or horizontal) and replicate the buffer.

with an argument, it will split the buffer and dump argument there. try ```:help split``` in vim.

whether or not this is possible in chrome is another question.
